### ISSUE TITLE
Enable turn off dimmer option

### ISSUE THREAD

**Issue by vinird** [vinird/tutorialjs#17]
Currently there is no option to disable the dimmer. In case the user wan to use only the popups we need to add a boolean option that prevent the dimmer to show up.

**Comment by takejaw6** [comment]
I will work on this too.

**Comment by adellahlou** [comment]
Putting the comment from my PR here, let me know what you think

I debated about how to implement this. tutorial.js has a critical path along which it calls functions, some of which manage the dimmer. Ultimately I ended up putting the if statement in each of the functions that manage the dimmer. To me, this accomplishes a policy-based design where the critical path stays untouched to act as a shell unaware of configuration, while behavior is the responsibility of the components.

I think from here there are two ways to go:
- Go one step further and move dimmer into its component aka own space/class
- Adopt a mixin/middleware architecture like express
